Directory.Build.props

```diff
@@ -6,4 +6,9 @@
     <MinimalWindowsVersion>10.0.19041.0</MinimalWindowsVersion>
     <WindowsSdkPackageVersion>10.0.22621.57</WindowsSdkPackageVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)\.editorconfig" />
+    <None Include="$(MSBuildThisFileDirectory)\Directory.Build.props" />
+    <None Include="$(MSBuildThisFileDirectory)\Directory.Packages.props" />
+  </ItemGroup>
```

Comment:
This will make them appear on all projects. Instead you should add them as solution items.
I would recommend holding up the change until we migrate to slnx.

Reply:
I'm aware of that

Reply:
IMO it's a better experience to add them to the project directly as they apply to every project
If it was an explicit `.props` file that was in the actual project structure then I would add them to the solution, though since these files apply everywhere it makes sense to put them inside every project as None.

Reply:
You can create a virtual folder and add them to it in the solution file.

Reply:
@Lamparter we've just migrated to slnx and can move forward with these changes now.

Reply:
Migrating to SLNX doesn't affect these changes.